pkgs/by-name/hu/hugo/package.nix

```diff
@@ -1,25 +1,25 @@
 { stdenv
 , lib
-, buildGoModule
+, buildGo123Module
```

Comment:
such overrides are preferred to be done via all-packages.nix

https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name#changing-implicit-attribute-defaults

Reply:
Got it, thank you. Now the version of the `go` package is already at `1.22.6` and above, so we can use `buildGoModule` again.